### PR TITLE
Release new version of submitit plugin

### PR DESCRIPTION
**Pull request by shagunsodhani**
## Motivation

Release new version of submitit plugin

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan

NA

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

#1358

### CODE REVIEW

File: plugins/hydra_submitit_launcher/setup.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
     LONG_DESC = fh.read()
     setup(
         name="hydra-submitit-launcher",
-        version="1.0.2",
+        version="1.2.0",
```
Comment (jieru-hu) [inline plugins/hydra_submitit_launcher/setup.py:9]: this is released off the master branch?  (what happens to 1.1.0?)

File: plugins/hydra_submitit_launcher/NEWS.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+1.1 (2021-01-30)
```
Comment (jieru-hu) [inline plugins/hydra_submitit_launcher/NEWS.md:1]: Could we add documentation on how to generate `NEWS.md` [here](https://hydra.cc/docs/next/development/documentation#news-entries) for both `core` and the plugins?

File: plugins/hydra_submitit_launcher/setup.py (hunk 1 above)
Comment (shagunsodhani) [reply plugins/hydra_submitit_launcher/setup.py:9]: 1.1.0 as the one released. We generally keep one version higher in setupy. (iirc)

File: plugins/hydra_submitit_launcher/NEWS.md (hunk 2 above)
Comment (shagunsodhani) [reply plugins/hydra_submitit_launcher/NEWS.md:1]: Probably this is a better [place](https://hydra.cc/docs/next/development/release) ?

File: plugins/hydra_submitit_launcher/setup.py (hunk 1 above)
Comment (jieru-hu) [reply plugins/hydra_submitit_launcher/setup.py:9]: yes you are right. sorry I got confused! Didn't realize the release already done and this is a cherry-pick :)

File: plugins/hydra_submitit_launcher/NEWS.md (hunk 2 above)
Comment (jieru-hu) [reply plugins/hydra_submitit_launcher/NEWS.md:1]: yes, this is better :)

**Review by jieru-hu — APPROVED** [review]

File: plugins/hydra_submitit_launcher/setup.py (hunk 1 above)
Comment (omry) [reply plugins/hydra_submitit_launcher/setup.py:9]: Did you release from the master branch?
submitit is probably still compatible, but we should get into the habit of releasing plugins from the appropriate branch.

Comment (shagunsodhani) [reply plugins/hydra_submitit_launcher/setup.py:9]: This is released from master. Earlier version was 1.0.2 . I locally bumped the version to 1.1 (before uploading to pypi). Then I bumped the version to 1.2 and made PR against master

Comment (omry) [reply plugins/hydra_submitit_launcher/setup.py:9]: Hydra in master is diverging from the 1.0 branch and there are already some differences in the examples:

```
$  cat plugins/hydra_submitit_launcher/example/config.yaml 
defaults:
  - override hydra/launcher: submitit_slurm

task: 1
```

In the future please release plugins from the appropriate branch.

**Review by omry — APPROVED** [review]
Can you merge this?

**Pull request merged**